### DISCUSSION TITLE
asciidoc support

### DISCUSSION THREAD

**Discussion by fekir** [GitJournal/GitJournal#539] [Feature Requests]
Github, Gitlab and other remotes support asciidoc files out of the box.

As ascidoc is better specified, and because there are multiple incompatible markdown variants (unsure which is supported by gitjournal), it would be a major feature if it would support asciidoc too.

It would be already nice if it where possible to edit/show `.adoc` file even without rendering them.

**Comment by fekir** [comment]
Hello,

as this issue has been closed without comments, and the latest gitjournal version (updated today) does not show the .adoc files, can I ask what's the status?

**Reply by vHanda** [reply]
Hey Fekir. I was trying to move all the feature requests to "GitHub discussions". Issues were voted on with the 👍 reactions, but discussions have their own voting mechanism. Moving an issue results in the reactions being moved, but it isn't easy to sort the discussions by reactions.

So in the end I'm not sure if I should move feature requests here or not.

I haven't made any progress on the ascii doc support

**Reply by fekir** [reply]
Ok, thank you for the explanation, I thought you decided to "reject" the issue with no further explanation.

>I haven't made any progress on the asciidoc support

I would be super-happy if currently gitjournal would show me that the file is there, and let me edit it.
Having it rendered would be of course better, but it is not as important.

For example, correct me if wrong, `.txt` files are not rendered.
For me it would be enough if gitjournal would handle `.adoc` files as `.txt`. (I cannot speak for the others that voted the issue, but think they would agree that having `.adoc` as `.txt` is better than nothing)

**Reply by vHanda** [reply]
Alright. This seems easier to do.

Actually, let me just make it configurable, that you can choose which files should be visible / hidden. I have other users asking for the ability to edit json/yaml files as well. The RawEditor would work for all these cases.

**Reply by vHanda** [reply]
Done: https://twitter.com/GitJournalApp/status/1449039758504697861

It'll be released with `1.82`,  probably sometime next week.

**Comment by aisbergde** [comment]
I also switched my notes System from Markdown to AsciiDoc. The syntax is so easy, like markdown, but there is one syntax and not a lot of dialects. And it supports include, and, and, and.

Unfortunately there is no AsciiDoc support on Android out of the box. There are some editors with AsciiDoc language support. To preview AsciiDoc content I need some tricky workflow: running a local Webserver, using a browser supporting desktop extensions, using the AsciiDoc extension to preview content.

Currently I use a mgit to put my notes under git version control. But the only way to use my AsciiDoc based notes system are editors.

So it would be really great, if Git Journal could support AsciiDoc. There are some ready technologies for preview AsciiDoc.

**Comment by aisbergde** [comment]
Is there anything we could contribute to implement some AsciiDoc support? For example I contributed AsciiDoc editing to Markor, and later someone else added AsciiDoc preview.
